exec/java-exec/src/main/resources/rest/profile/profile.ftl

```diff
@@ -162,8 +226,7 @@
       </div>
       <div id="fragment-overview" class="panel-collapse collapse">
         <div class="panel-body">
-          <svg id="fragment-overview-canvas" class="center-block"></svg>
```

Comment:
@arina-ielchiieva  is there a reason we don't display this canvas any more? Doesn't seem related to the list of bugs fixed.

Reply:
With this canvas, fragment overview was displayed weirdly. I have removed so the fragment overview looks the same as others.

Reply:
Talked to Kunal, it turned out that we need this canvas to display graphic for fragments execution time, so I guess we need to return it back.